geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaConsumerFeatureSource.scala

```diff
@@ -51,6 +50,10 @@ abstract class KafkaConsumerFeatureSource(entry: ContentEntry,
   def getReaderForFilter(f: Filter): FR
 }
 
+object KafkaConsumerFeatureSource {
+  lazy val wholeWorldBounds = ReferencedEnvelope.create(new Envelope(-180, 180, -90, 90), CRS_EPSG_4326)
```

Comment:
Yes!  In order to minimize depending on random jars in crazy places, this one-liner is repeated a few places.